### PR TITLE
Add the faq page

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
CC: @raukadah @palnabarun @sayanchowdhury

### CODE REVIEW

**Review by anistark — APPROVED** [review]

File: content/faq.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+### Q. Where can I find updates related to PyCon India 2021?
+
+All announcements related to the conference are posted to PyCon India mailing list.  You can receive the updates by joining [InPyCon mailing list](
+https://mail.python.org/mailman/listinfo/inpycon).
+
+You can also join our [Zulip channel](https://pyconindia.zulipchat.com/) or follow us on social media. All our announcements are posted on our [Twitter](https://twitter.com/pyconindia/), [LinkedIn](https://www.linkedin.com/company/pyconindia/), and [Facebook](https://www.facebook.com/PyConIndia/) pages.
+
+### Q. I want to present a talk/workshop at the conference. What should I do?
+
+You can submit your talk/workshop proposal on our [CFP Page](https://in.pycon.org/cfp/2020/proposals/). The deadline for submitting CFP for thsi year's conference has passed, please submit a talk next year.
```
Comment (anistark) [inline content/faq.md:20]: https://in.pycon.org/cfp/2021/proposals/

**Review by anistark — CHANGES_REQUESTED** [review]
Please fix the link. 2020 -> 2021

**Review by raukadah — APPROVED** [review]

**Review by vipulgupta2048 — COMMENTED** [review]
@Sc0rpi0n101 @raukadah Adding FAQ suggestions here, 
1. One about In-kind sponsors, I think we should highlight that here. (Question: How do I support the event?)
2. Pre-events and events leading up to the conference. Get the hype going, don't add anything specific just mention where the news will be (Twitter and mailing list) 
3. The combo ticket mention should list the exclusivity of combo tickets and a discount that we offer with them. 
4. Last FAQ be like: Still not convinced? Or got more questions? 
Answer: Watch the last year's conference (Youtube link) Or reach out to us on Zulip for any other queries. 
5. How is PyCon India being organized online? (People new to us may have doubts and we have a good answer. Give them the PPG to read and get more people wow'ed)

@Sc0rpi0n101 There are several spelling mistakes, pointed some out as after the PR is merged I can't add suggestions. 

- chek out the instructions
- tranfer your ticket(s) to someone else.
- submitting CFP for thsi year’s conference

Please run the content atleast aleast once through any grammar checker that you like.

**Comment by vipulgupta2048** [comment]
Closing https://github.com/pythonindia/inpycon2021-tasks/issues/69

**Pull request merged**